src/internal/packager/validate/validate.go

```diff
@@ -189,11 +191,11 @@ func validateComponent(pkg types.ZarfPackage, component types.ZarfComponent) err
 	return nil
 }
 
-func validateActionset(actions types.ZarfComponentActionSet) (bool, error) {
+func validateActionset(actionSet actions.ActionSet) (bool, error) {
 	containsVariables := false
 
```

Comment:
I decided to only pull actions.Validate() over - this logic is partly Zarf specific since it is looking for `variables` which only matters because Zarf has actions that don't support them (the rest of the logic is just simply looping through actions in the actionset)